admin/src/graphql/searchUsers.js

```diff
@@ -5,15 +5,13 @@ export const searchUsers = gql`
     $searchText: String!
     $currentPage: Int
     $pageSize: Int
-    $filterByActivated: Boolean
-    $filterByDeleted: Boolean
+    $filters: SearchUsersFiltersInput
```

Comment:
```suggestion
    $filters: SearchUsersFilters
```

Reply:
This is not possible, because `SearchUsersFilters` already exists as `ObjectType`.
And It's against the discribed global habit of naming `InputTypes` with `*Input` at the end. @Mogge